Artifacts/windows-powershellcore/installPowerShellCore.ps1

```diff
@@ -0,0 +1,79 @@
+﻿param
+(
+    [Parameter(Mandatory = $true)]
+    [String]
+    $packageUrl,
+
+    [Parameter()]
+    [bool]
+    $installCRuntime = $false
+)
+
+if (-not (Split-Path -Path $packageUrl -Leaf).EndsWith('.msi'))
```

Comment:
Would it not be sufficient to just check as follows? Why split the path?
```powershell
if (-not $packageUrl.EndsWith('.msi'))
```
You may, alternatively, want to add a specific check to ensure the URL is well-formed, as needed.